spring-boot-test-autoconfigure/src/test/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+# Use random port for embedded Mongod.
+spring.data.mongodb.port = 0
```

Comment:
No. But I don't know yet how to proceed. Can you join us on gitter when you have some time. We can chat about possible options. Thanks!